roles/db-start/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Ensure scripts directory is present
+  file:
+    path: "{{ scripts_dir }}"
+    state: directory
+    owner: "{{ oracle_user }}"
+    group: "{{ oracle_group }}"
+    mode: u=wrx,go=
+
+- name: Copy startup and shutdown scripts to target server
+  become: true
+  become_user: "{{ oracle_user }}"
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ scripts_dir }}/{{ item }}"
+    owner: "{{ oracle_user }}"
+    group: "{{ oracle_group }}"
+    mode: u=wrx,go=
+  with_items:
+    - "start_all.sh"
+    - "stop_all.sh"
+  tags: db-start
+
+- name: Setup automatic Oracle database stop and start services
```

Comment:
Please excuse my grammatical pet peeve here:  As a noun or adjectlve, it's "setup", but when used as a verb like here, it's "Set up"
https://langeek.co/en/grammar/course/1643/setup-vs-set-up